components/app/src/main/java/org/trellisldp/app/config/ResourceConfiguration.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.app.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ResourceConfiguration {
```

Comment:
This is too generic a name. Something that describes the one single resource configured would be appropriate-- `TriplestoreConfiguration` or the like.